C++/include/algorithm/sorting/README.md

```diff
@@ -0,0 +1,129 @@
+# Sorting
+Sorting algorithms are designed to arrange data in a particular order. Sorting algoritms have a few properties:
+- #### 1. Stable and Unstable Sorting:
+      Stable: identical elements remain in the same order as before sorting.
+      Unstable: identical elements may be sorted in any order.
+- #### 2. Internal and External Sorting:
+      Internal: Only the main memory of the computer or RAM is used. 
+      External: External memory like disk or tape is used.
+- #### 3. In-place and Not In-place Sorting:
+      In-place: where the elements are sorted by modifying their order directly.
+      Not In-place: where an auxiliary data structure is used for sorting.
+
+### Contents
+1. [Bubble Sort](#1-bubble-sort)
+2. [Selection Sort](#2-selection-sort)
+3. [Insertion Sort](#3-insertion-sort)
+1. [Quick Sort](#1-quick-sort)
```

Comment:
While simple numbered lists in Markdown can use this 1. 1. 1. formatting, this will not display correctly when you're using it in a table of contents. So just label them normally: 1, 2, 3, etc.